test/unit/io/sam_file/sam_file_format_test_template.hpp

```diff
@@ -32,9 +32,9 @@ using seqan3::operator""_tag;
 seqan3::alignment_file_input_options<seqan3::dna5> input_options;
 seqan3::alignment_file_output_options output_options;
 
-struct alignment_file_data : public ::testing::Test
+struct sam_file_data : public ::testing::Test
```

Comment:
Please document in a comment that this file/class is used for both sam and bam, which is not obvious any more since you changed the name.